.github/workflows/release-buildx.yml

```
@@ -8,35 +8,35 @@ on:
   workflow_dispatch:
     inputs:
       ref:
-        description: 'Ref (e.g. v0.10.0)'
+        description: 'Tag/ref to build (e.g. v0.10.0)'
```

```suggestion
        description: 'Tag or git reference to build from (e.g. v0.10.0)'
```

Can we provide a valid non-tag example? I think this would also accept;

- branch name
- maybe `refs/xx/xx/` (we could have a PR ref as example)
- git sha
- `git describe` format likely as well (e.g. `api/v1.52.0-beta.3-36-g5680867af8`)

---

I wouldn't encourage non-tag usage right now, as the VERSION string is derived from the reference.

So until we have an option to override the visible version (see https://github.com/docker/packaging/pull/260) I wouldn't want to provide other examples than tags.